util/src/main/java/io/kubernetes/client/util/PagerParams.java

```diff
@@ -0,0 +1,30 @@
+package io.kubernetes.client.util;
+
+public class PagerParams {
+
+  private Integer limit;
```

Comment:
nit, I prefer `int` to `Integer` but I don't feel strongly.

Reply:
i'm fine w/ either. a zero `int` and a null `Integer` actually works the same. 

[https://github.com/kubernetes/kubernetes/blob/d5a3db003916b1d33b503ccd2e4897e094d8af77/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L494-L497](https://github.com/kubernetes/kubernetes/blob/d5a3db003916b1d33b503ccd2e4897e094d8af77/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L494-L497)